.github/workflows/e2e.yml

```diff
@@ -98,15 +98,19 @@ jobs:
       run: |
         kubectl logs -l name=resource-topology -c resource-topology-exporter-container || :
 
-  e2e-metrics-https:
+  example_matrix:
```

Comment:
`e2e-metrics` perhaps? Or anything you like without "example" :)

Reply:
Yep, fixed it. Thanks!